docs/release-notes/v2.26/index.md

```diff
@@ -0,0 +1,64 @@
+# November.1 2020 (version 2.26)
+
+---
+
+Welcome to the November.1 2020 release of Gatsby (version 2.26). Key highlights of this release:
+
+- [New Image Plugin](#new-gatsby-image-beta) - access static images without GraphQL, high Lighthouse scores again
+- [File System Route API](#file-system-route-api) - create pages using filename conventions
+- [New Release Process](#new-release-process) - more stability and clarity around Gatsby releases
+- [gatsby-source-contentful v4.0](#gatsby-source-contentful) - new RichText implementation
```

Comment:
I would add the significant perf improvement for mdx sourcing (#27937 and if #27941 gets published then `lessBabel` will bring it even down further) here
Same for markdown (remark) but I don't think that'll be part of this release.

Reply:
Perf improvements are mentioned below and in the dedicated plugin section.